plugins/vocabbuilder.koplugin/main.lua

```diff
@@ -1498,31 +1512,38 @@ function VocabBuilder:addToMainMenu(menu_items)
                                 end
                             end
 
-                            builder_widget.current_lookup_word = item.word
+                            widget.current_lookup_word = item.word
                             self.ui:handleEvent(Event:new("LookupWord", item.word, true, nil, nil, nil, tweak_buttons_func))
                         end
                     })
                 end
                 return vocab_items
             end
 
-            self.builder_widget = VocabularyBuilderWidget:new{
+            self.widget = VocabularyBuilderWidget:new{
                 title = _("Vocabulary builder"),
                 select_items_callback = function(obj, start_idx, end_idx)
                     DB:select_items(obj, start_idx, end_idx)
                 end,
                 reload_items_callback = reload_items
             }
```

Comment:
Bit of cheating here with the indentation (from what I can notice with github web UI. I think these 9 lines should be shifted left.
It's small enough that all of it appearing in the diff should be ok.

Reply:
Done